specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -1097,17 +1097,1048 @@
           }
         }
       }
+    },
+    "/largepersongroups/{largePersonGroupId}/persons": {
+      "post": {
+        "description": "Create a new person in a specified large person group.",
+        "operationId": "LargePersonGroupPerson_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/largePersonGroupId"
+          },
+          {
+            "name": "body",
+            "description": "Request body for creating new person.",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/NameAndUserDataContract"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns a new personId created.",
+            "schema": {
+              "$ref": "#/definitions/Person"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create new person for large person group example": {
+            "$ref": "./examples/CreateNewLargePersonGroupPerson.json"
+          }
+        }
+      },
+      "get": {
+        "description": "List all persons in a large person group, and retrieve person information (including personId, name, userData and persistedFaceIds of registered faces of the person).",
+        "operationId": "LargePersonGroupPerson_List",
```

Comment:
If we prefer `LargeFaceList_ListFaces` then how about this? Besides, we also have `LargePersonGroupPerson_Create/Get/Update/Delete`.

Reply:
Using `LargeFaceList_ListFaces` is to be differentiated with `LargeFaceList_List` which refers to list large face lists. I guess you are thinking about the pattern of `LargeFaceListFace_xx` for face level operations. Yes, it's another pattern while my thought is the SDK could be more friendly to user if it uses same level as [API reference](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/5a158c10d2de3616c086f2d3), where LargeFaceList operations are all inside `LargeFaceList` section, without another level of `LargeFaceListFace` section.

Reply:
Actually I am think about using `LargePersonGroup_ListPersons` to align with `LargeFaceList_ListFaces`, but I think it is OK if we keep following the structure of API reference.